### DISCUSSION TITLE
taro小程序接入腾讯视频

### DISCUSSION THREAD

**Discussion by Hanchengluo** [NervJS/taro#14140] [Q & A] [answered]
### 这个特性解决了什么问题？
小程序不允许直接使用视频要使用视频插件，但taro找不到正常的接入腾讯视频方式。

### 这个 API 长什么样？
taro小程序正常接入腾讯视频

**Comment by ZakaryCode** [comment, marked as answer]
Taro 使用小程序插件[文档](https://nervjs.github.io/taro-docs/docs/hybrid#%E4%BD%BF%E7%94%A8%E5%B0%8F%E7%A8%8B%E5%BA%8F%E6%8F%92%E4%BB%B6)
小程序插件使用-腾讯视频插件[文档](https://developers.weixin.qq.com/community/develop/doc/0006823b3204681d28277dacd51004)

**Reply by Hanchengluo** [reply]
nutui react里找不到app.config.js和page.config.js
